src/PrestaShopBundle/Twig/Extension/GridExtension.php

```diff
@@ -168,8 +168,7 @@ public function isOrderingColumn(array $grid)
     {
         if (empty($grid['columns'])
             || empty($grid['sorting']['order_by'])
-            || empty($grid['sorting']['order_way'])
-            || 'asc' != strtolower($grid['sorting']['order_way'])) {
+            || empty($grid['sorting']['order_way'])) {
```

Comment:
I disable the condition that removed the drag-and-drop when position sorting direction was not ASC